python/pyarrow/scalar.pxi

```diff
@@ -1027,6 +1027,32 @@ cdef class ExtensionScalar(Scalar):
         return pyarrow_wrap_scalar(<shared_ptr[CScalar]> sp_scalar)
 
 
+cdef class FixedShapeTensorScalar(ExtensionScalar):
+    """
+    Concrete class for fixed shape tensor extension scalar.
+    """
+
+    def to_numpy_ndarray(self):
+        """
+        Convert fixed shape tensor extension scalar to a numpy.ndarray with zero copy.
+        """
+        return self.to_tensor().to_numpy()
+
+    def to_tensor(self):
+        """
+        Convert fixed shape tensor extension scalar to a pyarrow.Tensor.
```

Comment:
Can you add a Returns section? (then that will link to the pyarrow.Tensor docs I think)

Reply:
Done.

Reply:
Not done.

Reply:
Thanks for catching that! I must have gotten distracted by `FixedShapeTensorArray.to_tensor()`. Hopefully done now.